backend/src/charityprojects/api/views.py

```diff
@@ -283,19 +283,11 @@ def update_user_invitation(request):
         message = json_data["invitation_message"]
         project_user_id = ProjectUser.objects.filter(user_id=user_id, project_id_id=project_id)[0].id
         prize_given_id = ProjectUserDetails.objects.filter(pu_id_id=project_user_id)[0].prize_given_id_id
-        print(project_user_id)
-        print(prize_given_id)
         for email in invited_users:
-            invited_user = User.objects.get(email=email)
-            if invited_user:
-                invited_user_id = invited_user.id
-                invitation_date = date.today()
-                user_invitation = UserInvitation.objects.create(pu_id_id=project_user_id, friend_id=invited_user_id,
-                                                                status="Pending", invitation_message= message,
-                                                                prize_given_id_id=prize_given_id, invitation_date = invitation_date)
-                user_invitation.save()
+            if create_user_invitation(email, project_user_id, prize_given_id, message):
+                response["status"] = "Successfully stored invitation"
             else:
-                response = {'status': "Requested user does not exist"}
+                response["status"] = "Requested user does not exist"
```

Comment:
Invited users can be 10 users, lets say one of the user does not exist, this above line will set the complete status to "requested user does not exist". 

So I think either say status for each user is a list with their success or failure. 
Or set the success for the complete method.